site/viamguides/src/home-assistant/home-assistant.md

```diff
@@ -0,0 +1,190 @@
+author: HipsterBrown
+id: home-assistant
+summary: Add Viam to your Home Assistant with the community integration.
+categories: Developer
+environments: web
+status: Published 
+feedback link: https://github.com/viam-labs/viamcodelabs/issues
```

Comment:
The org updated to `viam-devrel`. Redirect works, but can create an issue to address in template builder and replace all.